front/src/modules/study/components/AddOrEditStudyModal.tsx

```diff
@@ -155,6 +155,19 @@ export default function AddOrEditStudyModal({ editionMode, study }: Props) {
     }
   }, [isCreateStudyError, isPatchStudyError, isDeleteStudyError]);
 
+  const isInvalidDate = () => {
+    if (currentStudy.start_date) {
+      const startDate = new Date(currentStudy.start_date);
+      if (currentStudy.expected_end_date && new Date(currentStudy.expected_end_date) < startDate) {
```

Comment:
You can use days.js for dates manipulation https://day.js.org/docs/en/query

Reply:
Didn't need it anymore since we can just compare the string values